pages/book/operators.mdx

```diff
@@ -287,16 +287,16 @@ Perform inequality and equality checks, find bigger, smaller or equal values.
 
 #### Not equal, `!=` [#binary-not-equal]
 
-Binary inequality (_not equal_) operator `!={:tact}` checks whether its two operands are _not equal_, returning a result of type [`Bool{:tact}`](/books/types#booleans). Doesn't perform implicit type conversions, always strict and requires both operands to be of the same type.
+Binary inequality (_not equal_) operator `!={:tact}` checks whether its two operands are _not equal_, returning a result of type [`Bool{:tact}`][bool]. Requires both operands to be of the same type and doesn't perform implicit type conversions, except for the [`Cell{:tact}`][p] and [`Slice{:tact}`][p] types, which are implicitly compared by their hashes.
 
-Can be applied to values of following types:
+Can be applied to [optionals](/book/optionals) and values of following types:
```

Comment:
I'd add `T?` to the list below (with the link to [optionals](https://docs.tact-lang.org/book/optionals)). It's easier to scan bullet points if you are using the book as a references